src/main/java/rs/teslaris/core/service/impl/document/DocumentPublicationServiceImpl.java

```diff
@@ -395,7 +396,14 @@ protected void setCommonFields(Document document, DocumentDTO documentDTO) {
         document.setScopusId(documentDTO.getScopusId());
 
         if (Objects.nonNull(documentDTO.getEventId())) {
-            document.setEvent(eventService.findEventById(documentDTO.getEventId()));
+            var event = eventService.findEventById(documentDTO.getEventId());
+
+            if (event.getSerialEvent()) {
```

Comment:
Usually for boolean properties the method is isSerialEvent() instead of getSerialEvent(). However, we should be consistent, if you are using get for boolean properties in other cases, we can keep that.

Reply:
This is a standard way lombok generated it. Should we keep it this way?

Reply:
let's than keep it